MAINTAINERS.md

```diff
@@ -0,0 +1,51 @@
+# Helm Maintainers
+
+This document explains the leadership structure of the Kubernetes Helm project, and list the current project maintainers.
+
+The Helm project welcomes contributions from any interested parties. This document explains the role of what we call "Core Maintainers". Core Maintainers play the special role of curating the project.
+
+## What is a Core Maintainer?
+
+(Unabashedly stolen from the [Docker](https://github.com/docker/docker/blob/master/MAINTAINERS) project)
+
+There are different types of maintainers, with different responsibilities, but
+all maintainers have 3 things in common:
+
+1. They share responsibility in the project's success.
+2. They have made a long-term, recurring time investment to improve the project.
+3. They spend that time doing whatever needs to be done, not necessarily what
+is the most interesting or fun.
+
+## Helm Core Maintainers
+
+The duties of a core maintainer include:
+
+* Classify and respond to GitHub issues and review pull requests
+* Perform code reviews (see below)
+* Shape the Helm roadmap and lead efforts to accomplish roadmap milestones
+* Participate actively in feature development and bug fixing
+* Answer questions and help users
+
+The current core maintainers of Helm (in alphabetical order):
+
+* Ville Aikas - PREFERRED EMAIL ([@vaikas-google](https://github.com/vaikas-google))
+* Matt Butcher - <butcher@deis.com> ([@technosophos](https://github.com/technosophos))
+* Jack Greenfield - PREFERRED EMAIL ([@jackgr](https://github.com/jackgr)) 
+* Brendan Melville - PREFERRED EMAIL ([@bmelville](https://github.com/bmelville))
+* Michelle Noorali - <michelle@deis.com> ([@michelleN](https://github.com/michelleN)]
+* Adam Reese - <areese@deis.com> ([@adamreese](https://github.com/adamreese))
+
+### Code Reviews
+
+Before being merged, every pull request must have at least two LGTMs ("Looks Good To Me"), and at least one of these must come from one of the core maintainers listed above.
```

Comment:
yes, requiring 2 can slow things down at awkward times (e.g., late at night before a release when only the 2 people on point for the release are still up at that hour).